spec/src/main/asciidoc/microprofile-opentracing-spec.asciidoc

```diff
@@ -163,6 +163,9 @@ Spans created for outgoing requests will have the following tags added by defaul
 
 Tags.SPAN_KIND MUST be specified at Span start time.
 
+An Tags.ERROR tag SHOULD be added to a Span on failed operations. It means for any client error (4xx) codes.
+If there is an exception object available the implementation SHOULD also add logs `event=error` and `error.object=<error object instance>` to the active span.
+
```

Comment:
The exception probably would never happen in this case. If we got a 4xx from the client, then the code was successfully executed. I don't think it hurts anything to leave the extra sentence in though.

Reply:
True, exceptions are not available in the client filter. However it might be implementation specific - e.g. tracing done directly in the apache HC